build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/JdkInfo.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Xamarin.Android.Build.Utilities;
+
+namespace Xamarin.Android.BuildTools.PrepTasks
+{
+	public class JdkInfo : Task
+	{
+		[Required]
+		public ITaskItem Output { get; set; }
+
+		public override bool Execute ()
+		{
+			Log.LogMessage (MessageImportance.Low, $"Task {nameof (JdkInfo)}");
+			Log.LogMessage (MessageImportance.Low, $"  {nameof (Output)}: {Output}");
+
+			AndroidLogger.Error += (task, message) => Log.LogError ($"{task}: {message}");
+			AndroidLogger.Warning += (task, message) => Log.LogWarning ($"{task}: {message}");
+			AndroidLogger.Info += (task, message) => Log.LogMessage (MessageImportance.Low, $"{task}: {message}");
+			AndroidSdk.Refresh ();
+
+			var javaSdkPath = AndroidSdk.JavaSdkPath;
+			Log.LogMessage (MessageImportance.Low, $"  {nameof (AndroidSdk.JavaSdkPath)}: {javaSdkPath}");
+
+			var jvmPath = Path.Combine (javaSdkPath, "jre", "bin", "server", "jvm.dll");
+			if (!File.Exists(jvmPath)) {
+				Log.LogError ($"JdkJvmPath not found at {jvmPath}");
+				return false;
+			}
+
+			var javaIncludePath = Path.Combine (javaSdkPath, "include");
+			var includes = new List<string> { javaIncludePath };
+			includes.AddRange (Directory.GetDirectories (javaIncludePath)); //Include dirs such as "win32"
+
+			var includeXmlTags = new StringBuilder ();
+			foreach (var include in includes) {
+				includeXmlTags.AppendLine ($"<JdkIncludePath Include=\"&quot;{include}&quot;\" />");
+			}
+
+			var directory = Path.GetDirectoryName (Output.ItemSpec);
+			if (!Directory.Exists (directory))
+				Directory.CreateDirectory (directory);
+
+			File.WriteAllText (Output.ItemSpec, $@"<Project xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"">
+  <Choose>
+    <When Condition="" '$(JdkJvmPath)' == '' "">
+      <PropertyGroup>
+        <JdkJvmPath>&quot;{jvmPath}&quot;</JdkJvmPath>
```

Comment:
Does this value actually need quoting? The "downstream" users of `$(JdkJvmPath)` don't already quote?

Actually, I can't find *any* use of `$(JdkJvmPath)`, which (1) suggests that perhaps we don't need it at all? and (2) means it *certainly* shouldn't be quoted here.